winit/src/application.rs

```diff
@@ -337,18 +337,22 @@ pub trait Application: Sized {
                 ..
             } => match window_event {
                 WindowEvent::Resized(new_size) => {
+                    size = Size::new(new_size, size.scale_factor());
+
                     events.push(Event::Window(window::Event::Resized {
-                        width: new_size.width.round() as u32,
-                        height: new_size.height.round() as u32,
+                        width: size.logical().width.round() as u32,
+                        height: size.logical().height.round() as u32,
                     }));
 
-                    size = new_size;
                     resized = true;
                 }
                 WindowEvent::CloseRequested => {
                     *control_flow = ControlFlow::Exit;
                 }
                 WindowEvent::CursorMoved { position, .. } => {
+                    let position =
+                        position.to_logical::<f64>(size.scale_factor());
+
                     events.push(Event::Mouse(mouse::Event::CursorMoved {
                         x: position.x as f32,
                         y: position.y as f32,
```

Comment:
Would is be desirable to mirror `winit` and use `f64` here?

Reply:
Probably, but we are currently using single precision almost everywhere.

I believe we need to think a bit more about it and eventually tackle it in a different PR.